tools/release/publish_release.sh

```diff
@@ -760,9 +782,9 @@ if [ "$cherry_pick_resolved" = "false" ]; then
             output=$(gh pr view $pr --json state,mergeCommit,baseRefName)
             state=$(echo $output | jq -r .state)
             commit=$(echo $output | jq -r .mergeCommit.oid)
-            target_branch=$(echo $output | jq -r .baseRefName)
-            echo -n "$pr $state $commit $target_branch:"
-            if [[ "$state" != "MERGED" || "$target_branch" != "main" ]]; then
+            target_pr_branch=$(echo $output | jq -r .baseRefName)
```

Comment:
this overloading was causing one of the issues where 'target_branch' was the rc-xxx branch but get's set back to 'main' here so just setting this to an explicit var to avoid that.